news/2024/2024-09-29-community-mentorship-program-autumn-2024-signups-now-open.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Community Mentorship Program Winter 2024 Signups Now Open
+date: 2024-09-29 10:00:00 +0000
+---
+
+The 28th cycle of the Community Mentorship program is upon us! Get ready to delve deep into the world of mapping, modding, or storyboarding.
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as a place for upcoming mappers, modders, and storyboarders to develop their skills under the guidance of more experienced members in the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.
```

Comment:
```suggestion
The Community Mentorship Program serves as a place for upcoming mappers, modders, and storyboarders to develop their skills under the guidance of more experienced members in the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.
```